website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -0,0 +1,141 @@
+
+To make the process as straightforward as possible, we've listed as much as we could to make a conference truly inclusive to our community but also included alternatives if a particular accommodation is not feasible. If you have any questions or concerns, please get in touch with us at cncf-tag-contributor-strategy@lists.cncf.io. 
+
+**Note**: This document is open source. If we missed anything, please submit a pull request or issue so we can update it — thank you! 
+
+## DEIA: Diversity + Equity + Inclusion + Accessibility
+
+We encourage you to include the A in your DEI efforts: **Diversity + Equity + Inclusion + Accessibility (DEIA)** — all too often is our community an afterthought. We want to change that and need your support!
+
+## ADA coordinator/specialist role
+
+We recommend appointing a designated Americans with Disabilities Act (ADA) coordinator to handle any prompt communications involving accommodations before and during the conference. This includes communication with attendees, speakers, facilities, and other event organizers.
+
+### Be ADA-compliant
+
+While nearly all buildings in the United States are ADA-compliant, we often see a gap at conferences: risers/stages generally don't have ramps, making them inaccessible for people with physical disabilities. Carpets make it harder for wheelchair users, physically and battery draining. Provide assistance when asked and have charging stations available.
+
+## Website, registration and CFP form, and program information
+
+### Event website
+
+On the registration and call for papers (CFP) pages, state that accommodations will be made for speakers and attendees with disabilities. Clearly and prominently stating that will make us feel welcome and motivate us to participate. Don't bury that important information in a registration form we might never see. Here's some suggested copy:
+
+> Interested in attending or speaking but require special accommodations? Please inform us through the registration or CFP form about your specific needs, and we will make every effort to provide the necessary support. For accommodation-related questions, please reach out to [accommodations@yourevent.com](mailto:accommodations@yourevent.com).
+
+Consider creating a dedicated email address (e.g., accommodations@) to signal you take this seriously.  
+
+### Registration and CFP form
+
+Include a field asking if accommodations are needed (yes/no) and a free-form text box to specify. Everyone's needs differ; it's hard to discern from a fixed form/dropdown.
+
+### Program Information
+
+On the program page, include session accessibility information, such as whether captioning will be provided and in which language. Also, mention that interpreters will be available upon request if requested in advance, and specify which sign language (e.g., American Sign Language, International Sign Language, etc.).  
```

Comment:
```suggestion
On the program page, include session accessibility information, such as whether captioning will be provided and in which language. Also, mention that interpreters will be available upon request if requested in advance, and specify which sign language (e.g., American Sign Language, International Sign Language, etc.).  Don't forget to include where and how interpreters are to be requested.
```

Reply:
Good point thank you.  I reqested that DHOH group make and issue and follow up with an addtional PR.